pkg/controller/perconaservermongodbbackup/perconaservermongodbbackup_controller.go

```diff
@@ -106,6 +106,9 @@ func (r *ReconcilePerconaServerMongoDBBackup) Reconcile(request reconcile.Reques
 	status := cr.Status
 
 	defer func() {
+		if cr.Status.State == psmdbv1.BackupStateReady {
```

Comment:
why it is inside defer?
I think we should not reconcile any CR which already has an error or ready state (like place the same code after 104 line

Reply:
https://github.com/percona/percona-server-mongodb-operator/pull/697/commits/a1108587aca5267e1c585320b73bdfa1c2d65799